src/constants/routes.js

```diff
@@ -167,4 +167,8 @@ export const modals = {
     isPrivate: true,
     forbiddenTokens: [tokenMap.BTC.key],
   },
+  delegatePerformance: {
+    isPrivate: false,
+    forbiddenTokens: [],
```

Comment:
Please add bitcoin to forbiddenTokens: `forbiddenTokens: [tokenMap.BTC.key],`

Reply:
Done